src/js/components/mapWidgets/widgetContent/searchContent.tsx

```diff
@@ -18,9 +18,13 @@ const SearchContent: FunctionComponent = () => {
   const [formError, setFormError] = useState(false);
   const [latitudeInput, setLatitude] = useState('');
   const [longitudeInput, setLongitude] = useState('');
-  const selectedLanguage = useSelector(
-    (state: RootState) => state.appState.selectedLanguage
+  const { selectedLanguage, selectedSearchWidgetLayer } = useSelector(
+    (state: RootState) => state.appState
   );
+  const [selectedLayerInfo, setSelectedLayerInfo] = useState({
+    displayField: '',
+    layerTitle: ''
+  });
```

Comment:
why do we need local state here if we are already getting this info from redux store?